lib/faraday/options/ssl_options.rb

```diff
@@ -12,7 +12,7 @@ module Faraday
   #   #           during the handshake or not (see https://github.com/ruby/openssl/pull/60)
   #   #
   #   # @!attribute hostname
-  #   #   @return [String] SNI hostname (see https://ruby.github.io/openssl/OpenSSL/SSL/SSLSocket.html#method-i-hostname-3D)
+  #   #   @return [String] Server hostname used for SNI (see https://ruby-doc.org/stdlib-2.5.1/libdoc/openssl/rdoc/OpenSSL/SSL/SSLSocket.html#method-i-hostname-3D)
```

Comment:
Hm, is that a very old Ruby docs link?

Reply:
Agreed.. was just being consistent within the file without expanding scope to change more.

The two files use different paths.

I'm not surprised you noticed as well given how small the pr is